src/lib/plot/line.typ

```diff
@@ -317,3 +317,103 @@
     plot-stroke: stroke
   ),)
 }
+
+/// Fill the area between two graphs. This behaves same as `plot` but takes
+/// a pair of data instead of a single data array/function.
+/// The area between both function plots gets filled.
+///
+/// This can be used to display an error-band of a function.
+///
+/// - domain (array): Domain tuple of the plot. If `data-a` or `data-b` is a function,
+///                   domain must be specified, as `data-a` or `data-b` is sampled
+///                   for x-values in `domain`. Values must be numbers.
```

Comment:
If domain is used in every function for the plot lib would it be worth defining a domain type somewhere?

Reply:
Yes, same for axes. :+1: